nx/source/runtime/devices/fs_dev.c

```diff
@@ -242,7 +242,7 @@ fsdev_getfspath(struct _reent *r,
     return -1;
 
   memset(outpath, 0, FS_MAX_PATH);
```

Comment:
If there's a memcpy below, I think this memset is redundant

Reply:
risks leaving unterminated string though.

`outpath[FS_MAX_PATH-1] = 0;`?

Reply:
nvm, there's explicit termination below as well